app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Post;
+use Illuminate\Support\Facades\Auth;
+
+class PostController extends Controller
+{
+
+    public function index()
+    {
+        if(Auth::guest()){
+            return redirect('/login');
+        }
+
+        $user_id = auth()->user()->id;
+        $posts = Post::where('user_id', $user_id)->get();
+        return view('posts.index')->with(['posts'=> $posts]);
+    }
+
+    public function create()
+    {
+        return view('posts.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $request->validate([
+            'title' => 'required',
+            'content' => 'required',
+        ]);
+         $re = Post::create($request->all());
```

Comment:
Use only ->validated, otherwise `slug` or other fields can be sent from UI
add user_id from controller by checking auth()